server/src/graphql/typedefs/pathway.graphql

```diff
@@ -10,6 +11,11 @@ type Content {
     content: String
 }
 
+type Tag {
```

Comment:
Could add an `id` field to `Tag` to maintain uniformity. We want each database entity to have a unique-throughout-the-db ID. We wanna ensure that tags are referenced at the back-end by their `id`s and not their names.

Could their names be their IDs? Maybe. But tags must be indexed on their name then, and names could contain spaces. Doesn't feel right, but this is pretty arbitrary. I'll leave the decision to you.

Reply:
Tag names are not supposed to contain spaces they’re supposed to be hyphenated like `react-js` or `computer-networks`. But an ID could be added too.

Reply:
On a second thought, having an additional `Id` field on Tags will be redundant and also make the `createUpdatePathway` mutation more complicated than necessary. 

Here’s the reasons: 

1. Querying a tag is always gonna be using the tag name. Because the only time a tag will get queried independently is when a user clicks on a tag. 

2. Having a client assigned ID forces the user to provide a new piece of information (whether the tag exists or not) rather if we just let the user give a tag name, we can use a `MERGE` query in Cypher to do the job.